reference/employer/openapi.yaml

```diff
@@ -518,6 +519,10 @@ paths:
                               type: boolean
                               description: '`true` if the individual an an active employee or contractor at the company.'
                               nullable: true
+                            class_code:
+                              type: string
+                              description: "Worker's compensation'classification code for this employee"
```

Comment:
I'm guessing this should be "Worker's compensation classification..." (no stray apostrophe)?

Reply:
@seii see #73